### PR TITLE
Pass hand type to correct behaviour during S4B1

### PR DESCRIPTION
**Pull request by TwelveNights**

### CODE REVIEW

File: imports/api/utils/NewGameUtils.js [hunk 1]
```diff
@@ -128,7 +128,8 @@ export default {
 		                          this.someoneAboveMinimum(Constants.JPN_END_POINTS);
 		// End condition where north player reaches first place after winning on last round
 		let dealerFirstAndAboveMinimum = Session.get("current_round") == 8 &&
-		                                 Session.get("current_bonus") > 0 &&
+										 Session.get("current_bonus") > 0 &&
```
Comment (0spooky) [inline imports/api/utils/NewGameUtils.js]: Change alignment to spaces so it doesn't mis-align

**Comment by 0spooky** [comment]
Has this been tested with the issue?  Just want to make sure.  Last push broke stuff.

**Comment by TwelveNights** [comment]
Yeah I tested it locally to see if it worked

**Comment by Futochu** [comment]
Looks good, can you reference issue #39 to link them?

File: imports/api/utils/NewGameUtils.js (hunk 1 above)
Comment (Futochu) [reply imports/api/utils/NewGameUtils.js]: You still haven't fixed this alignment issue.

File: imports/api/utils/NewGameUtils.js [hunk 2]
```diff
@@ -128,9 +128,10 @@ export default {
 		                          this.someoneAboveMinimum(Constants.JPN_END_POINTS);
 		// End condition where north player reaches first place after winning on last round
 		let dealerFirstAndAboveMinimum = Session.get("current_round") == 8 &&
-		                                 Session.get("current_bonus") > 0 &&
-		                                 this.getDirectionScore("north") >= Constants.JPN_END_POINTS &&
-		                                 this.getFirstPlace() == "north";
+					 		Session.get("current_bonus") > 0 &&
```
Comment (Futochu) [inline imports/api/utils/NewGameUtils.js]: Alignment is still wrong, please indent with tabs and align with spaces like how it was before.

**Pull request merged**